cmd/server-main.go

```diff
@@ -132,16 +132,15 @@ func serverMain(ctx *cli.Context) {
 		cli.ShowCommandHelpAndExit(ctx, "server", 1)
 	}
 
+	var quietMode bool
 	// Get "json" flag from command line argument and
-	// enable json and quite modes if jason flag is turned on.
-	jsonFlag := ctx.Bool("json") || ctx.GlobalBool("json")
-	if jsonFlag {
+	// enable json and quiet modes if json flag is turned on.
+	// Get quiet flag from command line argument, if json flag is not turned on.
+	if ctx.IsSet("json") || ctx.GlobalIsSet("json") {
+		quietMode = true
```

Comment:
This is a wrong approach.  You should not modify another variable here.

There is no change required other than renaming `*Bool()` to `*IsSet()`

Reply:
There is a bug here, whether json flag is set or quiet flag is set, the quietMode should be set to true

Reply:
That is why the variable name was changed from quietFlag to quietFlag, we are not really tracking whether quietFlag is set or not.

Reply:
What bug do you see?

 `json` and `quiet` flag are independent.  Its up to the logger to handle json/quiet differently or the same.

Reply:
I have added the comments in two places, where this variable is checked